pyproject.toml

```diff
@@ -47,10 +47,10 @@ classifiers = [
 dynamic = ["version"]
 
 dependencies = [
-    "anyio>=3.7.1,<4; python_version < '3.11'",
-    "anyio>=3.7.1,<5; python_version >= '3.11'",
+    "anyio>=3.7.1,<5",
```

Comment:
this will not work for Python bellow 3.11

Reply:
Why not? I tested locally and it works fine

Reply:
Btw, we was working with 4+ in the past, but with ExceptionGroup it brokes. For now your _compat works fine and the problem solved

Reply:
I tested locally with anyio version 5.x and Python 3.10 and it did not work

Reply:
But it is limitation for 3.X and 4.X only

Reply:
What problem did you have?

Reply:
from https://anyio.readthedocs.io/en/stable/versionhistory.html

4.0.0

BACKWARDS INCOMPATIBLE Replaced AnyIO’s own ExceptionGroup class with the PEP 654 BaseExceptionGroup and ExceptionGroup

BACKWARDS INCOMPATIBLE Changes to cancellation semantics:

- Any exceptions raising out of a task groups are now nested inside an ExceptionGroup (or BaseExceptionGroup if one or more BaseException were included)

- Fixed task group not raising a cancellation exception on asyncio at exit if no child tasks were spawned and an outer cancellation scope had been cancelled before

- Ensured that exiting a TaskGroup always hits a yield point, regardless of whether there are running child tasks to be waited on

- On asyncio, cancel scopes will defer cancelling tasks that are scheduled to resume with a finished future

- **On asyncio and Python 3.9/3.10, cancel scopes now only suppress cancellation exceptions if the cancel message matches the scope**

- Task groups on all backends now raise a single cancellation exception when an outer cancel scope is cancelled, and no exceptions other than cancellation exceptions are raised in the group

Reply:
It doesn't look like a change that could break our code.
Let me check again

Reply:
I had problems with it, but please check. Maybe I was doing something wrong.

Reply:
@davorrunje as you can see, specific test is passing too

Reply:
This [condition](https://github.com/airtai/faststream/blob/main/faststream/_compat.py#L198) provides us with a py3.11< compatibility
All other stuff worked correctly already, so there is no any problem to use anyio4.x with any python version